### DISCUSSION TITLE
Walrus WallPaper - Walrus 壁纸网站

### DISCUSSION THREAD

**Discussion by Heemale** [creatorsand/.github#109] [Ideas]
## 一句话 IDEA：

去中心化的壁纸网站

## 仓库地址

https://github.com/Heemale/walrus-wallpaper

## 我的角色：

前端开发、合约开发

## 社群昵称：

heemale

## 个人简介：

全栈开发

## 产品是给谁用的？

想要分享壁纸和下载壁纸的人

## 其他产品有什么痛点，用户为什么要选择你的产品？

**痛点：** 壁纸网站由中心化平台控制，用户无法保证自己上传的壁纸是否能永久存储或者是否被删除。
**解决方案：** 壁纸存储在去中心化网络中，用户的内容不受单一平台控制，能够保证数据的长久保存和不可篡改性。即使平台停止运营，壁纸也可以继续存在于区块链上。

## 主要功能
- 注册使用空间
- 上传分享壁纸
- 下载壁纸

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
